backend/FwLite/LocalWebApp/LocalWebApp.csproj

```diff
@@ -47,4 +47,9 @@
         <EmbeddedResource Include="..\..\..\frontend\viewer\dist\**\*" />
     </ItemGroup>
 
+    <ItemGroup>
+        <Content Include="/opt/local/lib/libicu*.??.dylib" Condition="$([MSBuild]::IsOsPlatform('macOS'))">
```

Comment:
This assumes that icu4c libs are installed under "/opt/local/lib" on macOS. This is the default location for libs that MacPorts installs. If we need to start bundling icu4c libs from somewhere other than "/opt/local/lib", this path and some other paths in the build scripts will need to be updated. We have to bundle the icu4c libs somehow unless macOS starts to include them by default.

Reply:
It would be good to put a comment in the code explaining that /opt/local/lib is where MacPorts puts the icu4c libraries, and also mentioning where Homebrew (https://brew.sh, `brew install icu4c`) puts them (probably in `/usr/local/lib` but I'd need to confirm that by installing on an actual Mac). Then you could actually write two `<Content` lines, each with a Condition clause to check if the file exists; one for `/usr/local/lib` and one for `/opt/local/lib`. That way we'd have both MacPorts and Homebrew covered.